src/comment_worker.py

```diff
@@ -685,7 +694,93 @@ def promote(self, sess, comment, investor, to_promote):
                 REDDIT.subreddit(subreddit).flair.set(user.name, f"{firm.name} | {flair_role_user}")
                 REDDIT.subreddit(subreddit).flair.set(investor.name, f"{firm.name} | {flair_role_investor}")
 
-        return comment.reply_wrap(message.modify_promote(user))
+        return comment.reply_wrap(message.modify_promote(user, user_role))
+
+    @req_user
+    def demote(self, sess, comment, investor, to_demote):
+        if investor.firm == 0:
+            return comment.reply_wrap(message.firm_none_org)
+
+        user = sess.query(Investor).\
+            filter(func.lower(Investor.name) == func.lower(to_demote)).\
+            first()
+        if (user is None) or (user.name == investor.name) or (user.firm != investor.firm):
+            return comment.reply_wrap(message.demote_failure_org)
+
+        firm = sess.query(Firm).\
+            filter(Firm.id == user.firm).\
+            first()
+
+        user_role = user.firm_role
+
+        if user_role == "":
+            return comment.reply_wrap(message.demote_failure_trader_org)
+        elif user_role == "assoc":
```

Comment:
Line 718, 725, 737, 749 you have `elif` statements whan you can have just a simple `if` statement.

By the logic, you don't have a final set of instructions to execute, so all `elif`s can be changed into if.